spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/hazelcast/HazelcastConfigResourceCondition.java

```diff
@@ -43,6 +46,16 @@ protected HazelcastConfigResourceCondition(String configSystemProperty, String..
 		this.configSystemProperty = configSystemProperty;
 	}
 
+	@Override
+	public ConditionOutcome getMatchOutcome(ConditionContext context, AnnotatedTypeMetadata metadata) {
```

Comment:
This code doesn't match with the javadoc or name of `HazelcastConfigResourceCondition` so it shouldn't be located there.